hack/release.toml

```
@@ -243,6 +243,36 @@ Talos now uses `registry.k8s.io` instead of `k8s.gcr.io` for Kubernetes containe
 See [Kubernetes documentation](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/) for additional details.
 
 If using registry mirrors, or in air-gapped installations you may need to update your configuration.
+"""
+
+    [notes.registry-mirrors]
+        title = "Registry Mirrors"
+        description = """\
+Talos had a inconsistency in the way registry mirror endpoints are handled when compared with `containerd` implementation:
```

```suggestion
Talos had an inconsistency in the way registry mirror endpoints are handled when compared with `containerd` implementation:
```

I think. Right @TimJones ?

---

Yup! I missed that one!